lib/ctws.rb

```diff
@@ -3,14 +3,15 @@
 module Ctws
   mattr_accessor :user_class
   mattr_accessor :user_class_strong_params
-  mattr_accessor :user_validate_with_password
   mattr_accessor :jwt_expiration_time
   mattr_accessor :jwt_auth_token_attrs
+  # if nil or false user password won't be checked
```

Comment:
maybe we can add "`user_authentication_callback` will be callend with `emai` and `password` arguments.".

Reply:
yeah, `password` should also be checked, not only email

Reply:
but keep in mind the docs (README.md), where is stated that the password is optional:

> The **`password` is optional** by default a user is validated with password ...

So, I think the _opt-out-password_ feature should continue and not removed